docs/admin/eventing/broker-configuration.md

```diff
@@ -112,15 +112,23 @@ When a Broker is created without a specified `BrokerClass` annotation, the defau
 
 The following example creates a Broker called `default` in the default namespace, and uses `MTChannelBasedBroker` as the implementation:
 
-```bash
-kubectl create -f - <<EOF
-apiVersion: eventing.knative.dev/v1
-kind: Broker
-metadata:
-  name: default
-  namespace: default
-EOF
-```
+1. Create a YAML file for your Broker using the example below:
+
+    ```yaml
+    apiVersion: eventing.knative.dev/v1
+    kind: Broker
+    metadata:
+      name: default
+      namespace: default
+    ```
+
+1. Apply the YAML file by running the command:
+
+    ```bash
+    kubectl apply --filename <filename>.yaml
```

Comment:
```suggestion
    kubectl apply -f <filename>.yaml
```
I'd prefer we use just `-f` across the board and shorten the commands generally where possible. We'd need to update it in the style guide though (https://dev-knative.netlify.app/development/help/contributor/style-guide/documenting-code/#documenting-yaml)

Wdyt @RichardJJG @snneji ?

Reply:
I agree. I can make those changes.